bundles/org.openhab.binding.wundergroundupdatereceiver/src/main/java/org/openhab/binding/wundergroundupdatereceiver/internal/WundergroundUpdateReceiverHandler.java

```diff
@@ -60,6 +62,9 @@ public String getStationId() {
         return config.stationId;
     }
 
+    // The format can be "yyyy-MM-dd H:mm:ss" from the device
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd H:mm:ss");
```

Comment:
This format is different to the current one:
`yyyy-MM-dd H:mm:ss`
vs
`yyyy-MM-dd hh:mm:ss`

Just a double check, this intentional?

Reply:
Yes it is. My unit is sending things like "2025-02-07 9:04:03" in the datutc parameter, which the ISO formatter throws on.
Even the try catch-itself was just defensive programming from the start, so I hadn't noticed.
Unless there was a change in the formatter between java 11 and 21.